### ISSUE TITLE
Fails if message has field named `from`.

### ISSUE THREAD

**Issue by AudriusButkevicius** [nelfin/pylint-protobuf#46]
Please skip to the TL;DR if you don't care about the process of discovery.

So I was debugging why the plugin wasn't working, and arrived at the conclusion that the class def that the plugin adds properties onto to signal that it's a proto class, and the class def that gets passed to the tree walker end up being different instances.

I couldn't understand why that would be the case, but I took the warning as the cause:
```
batch/foo.py:1:0: E0611: No name 'protocol_pb2' in module 'repro.protocol' (no-name-in-module)
```
as the cause.

So I started debugging pylint, as to why pylint fails to import/parse the protobuf file.

I've applied the following diff to my pylint (on top of git+https://github.com/PyCQA/pylint.git@85ef625ed86ae584261f9e2dd1c7092db92b5da1):

```diff
--- ../ver.py	2021-04-05 00:05:27.659671359 +0100
+++ /home/audriusb/.pyenv/versions/3.7.9/lib/python3.7/site-packages/pylint/checkers/variables.py	2021-04-05 00:06:35.584358947 +0100
@@ -1926,16 +1926,22 @@
                 module = None
                 break
             try:
+                print("before", module and module.name)
                 module = next(module.getattr(name)[0].infer())
                 if module is astroid.Uninferable:
                     return None
             except astroid.NotFoundError:
-                if module.name in self._ignored_modules:
+                print("after", module and module.name)
+                module = next(module.getattr(name)[0].infer())
+                if not module:
+                    if module.name in self._ignored_modules:
+                        return None
+                    self.add_message(
+                        "no-name-in-module", args=(name, module.name), node=node
+                    )
                     return None
-                self.add_message(
-                    "no-name-in-module", args=(name, module.name), node=node
-                )
-                return None
+                else:
+                    print("after second attempt", module and module.name)
             except astroid.InferenceError:
                 return None
         if module_names:
```

and I get the following output when running with the plugin:
```
/tmp/pylint2/repro $ python -m pylint repro.batch.foo -E --load-plugins=pylint_protobuf 
before repro
before repro.analytic
after repro.analytic
after second attempt repro.analytic.analytic_pb2
before repro.analytic.analytic_pb2
************* Module repro.batch.foo
batch/foo.py:8:8: E1101: Instance of 'StreamConfiguration' has no 'client' member (no-member)
```
I guess the plugin still doesn't work, but atleast the failure to load things is gone.
I suspect "if it failed try again" ends up invalidating/clearing things and the attributes the plugin set are gone, hence the plugin ends up not working.

Then, completely by accident, while building a repro for pylint, I ran pylint with the same debug code but without the protobuf plugin, and low-and-behold, the import error is gone:
```
/tmp/pylint2/repro$ python -m pylint repro.batch.foo -E
before repro
before repro.analytic
before repro.analytic.analytic_pb2
************* Module repro.batch.foo
batch/foo.py:8:8: E1101: Instance of 'StreamConfiguration' has no 'analytic' member (no-member)
```

Then I started trying to understanding what type of message was tripping the plugin, and effectively by binary searching.

TL;DR:

I've discovered that if the field is named "from", the plugin blows up preventing pylint from importing the module properly (and if being reimported, losing the state the plugin attached to class definitions.

**Comment by nelfin** [comment]
Hi @AudriusButkevicius, thanks for the report. I'm pretty sure you'd see this behaviour for messages with any field name matching a Python keyword

**Comment by nelfin** [comment]
Please check that `pip install git+https://github.com/nelfin/pylint-protobuf@74a11e4ff1a81a3d25732bc87c1ef9547f4e21f2` fixes your use-case

**Comment by AudriusButkevicius** [comment]
This fixes it. Thanks.

**Comment by nelfin** [comment]
Cool. I'm preparing for a v0.20.0 release later this week and this fix should go out then.
